notebooks/official/model_evaluation/automl_text_classification_model_evaluation.ipynb

```diff
@@ -0,0 +1,1706 @@
+{
```

Comment:
<p>why not just use the <code>model</code> object instead of the static class  <code>obs.BatchPredictionJob(</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/959/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
Since its just to check the state of BatchPredictionJob , we are using jobs.BatchPredictionJob instead of model object